astropy/io/fits/hdu/compressed/tests/test_tiled_compression.py

```diff
@@ -61,8 +67,12 @@ def test_zblank_support(canonical_data_base_path, tmp_path):
 
     hdu.writeto(tmp_path / "test_zblank.fits")
 
+    with fits.open(tmp_path / "test_zblank.fits", disable_image_compression=True) as hduc:
+        # The ZBLANK value is different from before as we don't preserve it,
+        # instead we always write out with the default ZBLANK
+        assert hduc[1].header['ZBLANK'] == -2147483648
```

Comment:
This PR doesn't actually change the behavior, but just updated the test to make it clear that before and after this PR, we didn't preserve ZBLANK values that were in existing headers - we only ever write out files with the default ZBLANK (there is no mechanism for using a custom ZBLANK). We can always revisit this in a future PR if we want, but just wanted to clarify what is happening here.